### ISSUE TITLE
weird unwanted highlight word on every top screen

### ISSUE THREAD

**Issue by myme5261314** [jeaye/color_coded#65]
Here's the thing, after carefully examination which plugin had caused the following issues as below screenshots indicated, finally I find it was **color_coded** which brings me that weird highlight.

Does anyone know how this happened and how to remove this highlight feature?

a.  After used color_coded.
> Plugin 'jeaye/color_coded'
> ![image](https://cloud.githubusercontent.com/assets/1814831/10013454/3a134d9c-6144-11e5-9236-b62692df91c9.png)
> ![image](https://cloud.githubusercontent.com/assets/1814831/10013483/695eac2c-6144-11e5-887d-9351c6f3b87f.png)
> ![image](https://cloud.githubusercontent.com/assets/1814831/10013522/acd17b56-6144-11e5-9a59-26df79c4a760.png)
> ![image](https://cloud.githubusercontent.com/assets/1814831/10013526/b46e076c-6144-11e5-8247-17ef92d4f92c.png)

b. Before used color_coded.
> " Plugin 'jeaye/color_coded'
> ![image](https://cloud.githubusercontent.com/assets/1814831/10013614/75e52a1a-6145-11e5-8c36-d4d3a2f278c2.png)
> ![image](https://cloud.githubusercontent.com/assets/1814831/10013635/a9f57756-6145-11e5-8d7c-7fab07da8d5c.png)
> ![image](https://cloud.githubusercontent.com/assets/1814831/10013663/f63bbb02-6145-11e5-955e-23b4df849dc0.png)
> ![image](https://cloud.githubusercontent.com/assets/1814831/10013678/11f09bf6-6146-11e5-9a46-57cdc53bfd28.png)

**Comment by jeaye** [comment]
@myme5261314 Sorry, I don't see the issue here. In your "after color_coded" shots, a couple of things are highlighted.

1. `VideoFrameRef` is highlighted and it looks like that's because you have `:set hls` enabled; use `:set nohls` to disable it; **this is not related to color_coded**
2. The various full-caps defines are highlighted, such as `MY_HEIGHT`, which makes sense, since you're using color_coded to highlight your code, I imagine.

So we can be 100% clear, are **you** 100% what color_coded is supposed to be doing? You're meant to supply it some configuration in a `.color_coded` file and it will use that information to compile your code, as you type, and change vim's highlighting to match the semantics of the code. From how it looks, you're using color_coded and you've **not** provided a valid `.color_coded` file, since not many things are actually highlighted.

Please let me know if you have further questions.

**Comment by myme5261314** [comment]
I think you have misunderstood my current issue. Maybe a gif record could help the illustration. The weird highlight always take place when the **screen scrolls down**.
> [gif record](http://showterm.io/d6ee99825af028d0a4516)

As refer to #63 , using absolute path will give me this.
> -x
c++
-DDEBUG
-I/home/deeplearn/library/caffe/include
-I/usr/include/openni2
-I/usr/local/cuda/include
-I/usr/local/include
-I/**project_path**/include
-Wall
-Werror
> ![image](https://cloud.githubusercontent.com/assets/1814831/10041880/5ac8688e-621a-11e5-9f06-7083e677e32c.png)

Using relative path gives me worse highlighting since it couldn't find header file.
> -x
c++
-DDEBUG
-I/home/deeplearn/library/caffe/include
-I/usr/include/openni2
-I/usr/local/cuda/include
-I/usr/local/include
**-Iinclude**
-Wall
-Werror
> ![image](https://cloud.githubusercontent.com/assets/1814831/10042134/0ed71b62-621c-11e5-9cc9-74db47128788.png)

**Comment by jeaye** [comment]
I've pushed up some big changes into master which fixes a number of highlighting issues. Let me know how it goes for you.

**Comment by jeaye** [comment]
@myme5261314 Any update?

**Comment by myme5261314** [comment]
Nope, but I found the issue really happens once I put the cursor to the place of left bracket `(` or right bracket `)`. When the cursor leave that position, the weird highlighted word disappeared. 

**Please notice the different cursor in normal mode (not mouse cursor I mean, I was putting the cursor at the left bracket in the first image, and `i` of `int` in the second image) and the weird highlighted word std.**

>PS: What else plugin of vim which I installed and related to the brackets is [wildfire.vim](https://github.com/gcmt/wildfire.vim). Maybe it's the conflicts between color_coded and this plugin?

------
> ![image](https://cloud.githubusercontent.com/assets/1814831/11620474/3c75c688-9cee-11e5-8ba0-086acf557192.png)

------

> ![image](https://cloud.githubusercontent.com/assets/1814831/11620481/52ca5f0c-9cee-11e5-95e5-66c6d7f79174.png)
